index.js

```diff
@@ -60,19 +60,40 @@ const Resizer = ({ array, updateCallback }) => {
   `;
 };
 
+let mouseDown = false;
```

Comment:
Although this hack works, it's really bad practice to use global variables, especially with React Apps.

If you needed to store these variables, you would convert either use React's useState hook, or convert this functional component to a `extends Component` class, so you can use `this.setState()`.

Turns out they aren't needed though, see my comments below.